tests/test_cat.py

```diff
@@ -576,14 +650,44 @@ def test_specsuffix_lost(self):
         :return:
         """
         statuses = [
-            ["newborn", 0, "kit"],
-            ["kitten", 1, "kit"],
-            ["apprentice", 6, "paw"],
-            ["warrior", 14, "test"],
+            [{"rank": CatRank.NEWBORN}, 0, "kit"],
+            [{"rank": CatRank.KITTEN}, 1, "kit"],
+            [{"rank": CatRank.APPRENTICE}, 6, "paw"],
+            [{"rank": CatRank.WARRIOR}, 14, "test"],
         ]
         for status, moons, suffix in statuses:
-            with self.subTest("lost clancats", status=status):
-                cat = Cat(status=status, moons=moons, suffix="test")
-                cat.outside = True
+            with self.subTest("lost clancats", status_dict=status):
+                cat = Cat(status_dict=status, moons=moons, suffix="test")
+                cat.status.become_lost()
                 cat.name.specsuffix_hidden = True
                 self.assertTrue(str(cat.name).endswith("test"))
+
+
+class TestSocialAssignment(unittest.TestCase):
+    def test_clancat_social(self):
+        clancat_ranks = (
+            CatRank.NEWBORN,
+            CatRank.KITTEN,
+            CatRank.APPRENTICE,
+            CatRank.MEDIATOR_APPRENTICE,
+            CatRank.MEDICINE_APPRENTICE,
+            CatRank.MEDICINE_CAT,
+            CatRank.MEDIATOR,
+            CatRank.DEPUTY,
+            CatRank.LEADER,
+            CatRank.ELDER,
+        )
+
+        for rank in clancat_ranks:
+            with self.subTest("clancat social assignment"):
```

Comment:
```suggestion
            with self.subTest("clancat social assignment", rank=rank):
```
putting this here tells us which rank in the subtest failed